cmake/FindASANDylib.cmake

```diff
@@ -14,8 +14,21 @@
 
 # Finds `libclang_rt.asan_osx_dynamic.dylib` on Apple platform, it is typically
 # under ${CMAKE_CXX_COMPILER}/../lib.
```

Comment:
Can you expand the comment to explain why finding the `dylib` file is necessary?

Reply:
Done.